The JJOB and yaml files for H(x) of the ensemble are generated correctly

flampouris opened the pull request:
Guillaume, if you work, please check this PR. 

It should not break anything on the 3Dvar case.

Commit:
The JJOB and yaml files for H(x) of the ensemble are generated correctly

guillaumevernieres committed:
Minor update and bugfix

Added ./ for users who don't have it in their path.
Instruction for rocoto check status was wrong.

guillaumevernieres commented on jobs/.gitignore:
```diff
@@ -0,0 +1,2 @@
+JJOB_PREP_LETKF?
+JJOB_PREP_LETKF??
```
That's a weird thing to have in the .gitignore!

flampouris commented:
I know, we will address the issue here:
https://github.com/NOAA-EMC/godas/issues/24

it's the jjobs for each member.



On Sun, Sep 8, 2019 at 1:04 PM guillaumevernieres <notifications@github.com>
wrote:

> *@guillaumevernieres* commented on this pull request.
> ------------------------------
>
> In jobs/.gitignore
> <https://github.com/NOAA-EMC/godas/pull/25#discussion_r322019416>:
>
> > @@ -0,0 +1,2 @@
> +JJOB_PREP_LETKF?
> +JJOB_PREP_LETKF??
>
> That's a weird thing to have in the .gitignore!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/godas/pull/25?email_source=notifications&email_token=AB7DH37YL2C45L4PTJF376LQIUWAVA5CNFSM4IURIXUKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCEADFTY#pullrequestreview-285225679>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB7DH32MBMLVOI4WMUA6N23QIUWAVANCNFSM4IURIXUA>
> .
>


-- 
Dr. Rer. Nat. Stylianos Flampouris
Team Leader for Marine Data Assimilation
IMSG at NOAA​​/NWS/NCEP/EMC
*​P: *+1(301)683-3789​ *E: *stylianos.flampouris@noaa.gov

guillaumevernieres commented:
@flampouris OK. Will look at it tomorrow.

jkuang-exploro committed:
Feature/jjob (#27)

* clean up, add comments

* more comments

* update case file

* remove user yaml

* remove user yaml

* add hera platform

* change bigmem tasks to shared in 3dvar and hybrid

Commit:
The develop #27 had two issues:
1. Indicators of confict from merging (<<<<<HEAD ) at the places.yaml
were removed
2. The CHGRP_RSTPROD_COMMAND is required and it was not included at
the initial hera.yaml; the CHGRP_RSTPROD_COMMAND was added.

Commit:
Load the correct jedi module

Commit:
Merge pull request #29 from NOAA-EMC/bugfix/issue28

The develop #27 had two issues.

guillaumevernieres committed:
Update to readme

guillaumevernieres committed:
Cosmetic changes

Commit:
The places.yml points to soca-static, all the members should have writing access to it (#34)

guillaumevernieres committed:
More cosmetic work

Commit:
Checking if the ADT IODA obs exist and prepares the obs for ingestion (#36)

* Checking if the ADT IODA obs exist and coppies them to the Rrundir/Data
If the ADT IODA obs do not exist but the raw data exist, it processes them and dumping them at the rundir/data

* Add the check of existing observations for the FNMOC
Issue #26

* Fixed wrong obs link

guillaumevernieres committed:
Update README.md

guillaumevernieres committed:
Updates to work on hera

Commit:
delete slash

Commit:
Merge pull request #33 from NOAA-EMC/feature/update-readme-gv

Update to readme

Commit:
Update README.md

guillaumevernieres committed:
fixed hera related issues

Commit:
Merge pull request #45 from NOAA-EMC/feature/soca-3dvar-hera

3DVAR cycling on Hera

jkuang-exploro committed:
fix the resource settings (#51)

guillaumevernieres committed:
Only 200 cores/10 nodes needed (possibly only 5 nodes)

guillaumevernieres committed:
Removed soca-bundle from the list of submodules

guillaumevernieres committed:
Updated the clone/build process in the readme

Commit:
Merge pull request #52 from NOAA-EMC/bugfix/wrong-resources

Wrong resources assigned to var

Commit:
Merge pull request #56 from NOAA-EMC/bugfix/remove-soca-bundle

Remove the soca-bundle from the list of submodules

guillaumevernieres committed:
update readme to be consistent with #56 (#57)

* update readme to be consistent with #56

* 2 day cycling

* Addressing reviewer's comments

* Update the README to allocate resources for the ctest.

* Create workflow directory

* Updated README.md

* wip

Commit:
FIX_SCRUB option for users

jkuang-exploro committed:
Revert "FIX_SCRUB option for users" (#67)

This reverts commit 671bca00382ea09f0b853051994c9926cfd17114.

jkuang-exploro commented:
This PR needs a merge from develop.

jkuang-exploro commented:
@flampouris I've tried merge develop into this PR but unfortunately you have to be the one to merge it with the develop since I lack the knowledge of the conflicts in JJOB_DA_PREP

jkuang-exploro committed:
Feature/pr ens (#65)

* letkf_only

* resource fix for final job ( service task )

* FIX_SCRUB option

* FIX_SCRUB bug fix

* fix letkf_only.yaml

flampouris commented:
I am working on it.

Commit:
PR for cleaning the README (#69)

* 1. Remove the recursive cloning to avoid issues with the submodules.
In principle, by running `git submodule update --init --recursive` is sufficient.

* 2. Update the README.md, to make more clear the cloning, deployment and running GODAS.

* 2.1 Update the "\" to fix the syntax.

Commit:
rebase

Commit:
Resolving Conflicts during rebasing

Commit:
Adding the case file for the letkf

Commit:
Update the README.md to include instructions for the letkf

Commit:
fix for ensemble rocoto

Commit:
Harmonize the LETKF scripts with the 3dvar

flampouris commented:
Holding on this PR, resolving some optimization issues first #73
